Scripts/build-phases/replace_secrets.rb

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env ruby
+##
```

Comment:
@jleandroperez – WDYT about skipping this and storing the file in mobile secrets as a `.swift` file rather than as JSON? It makes this script unnecessary as we can just copy the file directly without having the extra translation step.

Reply:
I mostly replicated the WooCommerce mechanism... which was implemented (**i think**) before the mobile-secrets repository was implemented.

And honestly, didn't even realize things can be a million times simpler. **Thank you** for mentioning this!!